tests/Feature/TeamInvitationEmailNormalizationTest.php

```
@@ -0,0 +1,63 @@
+            'link' => 'https://example.com/invite/test-uuid-123',
+            'via' => 'link'
+        ]);
+
+        // Verify the invitation email matches the user email (both normalized)
+        $this->assertEquals($user->email, $invitation->email);
+
+        // Verify user lookup works
+        $foundUser = User::whereEmail($invitation->email)->first();
+        $this->assertNotNull($foundUser);
+        $this->assertEquals($user->id, $foundUser->id);
+    }
```

PHPMD (2.15.0)

The method test_team_invitation_works_with_existing_user_email is not named in camelCase. (undefined)

(CamelCaseMethodName)